appConfig/appConfig.snd3.yaml

```diff
@@ -26,9 +26,9 @@
 - key: DEFRA_ID_POLICY
   value: "b2c_1a_cui_cpdev_signupsigninsfi"
 - key: DEFRA_ID_REDIRECT_URL
-  value: "https://ffc-sfd-landing-page.snd3.adp.defra.gov.uk/auth/sign-in-oidc"
+  value: "https://ffc-sfd.snd3.adp.defra.gov.uk/auth/sign-in-oidc"
```

Comment:
Couldn't remember if removing `landing-page` from the URLs in the `appConfig` also applied to these two Defra ID URLs or if what only applicable to the `GATEWAY_HOST`? Let me know and I can undo these changes if I'm incorrect.